### PR TITLE
[MTDSA-194] [DM] Add sandbox implementation of create/read liability

### PR DESCRIPTION
**Pull request by danmassie**
Provides the sandbox implementation of create and read liability giving static responses.

### CODE REVIEW

**Comment by dabd** [comment]
👍

File: resources/public/api/definition.json [hunk 1]
```diff
@@ -43,6 +43,22 @@
             "authType":"USER",
             "throttlingTier":"UNLIMITED",
             "scope": "read:self-assessment-api"
+          },
+          {
+            "uriPattern":"/{utr}/liabilities",
+            "endpointName":"Request Liability",
+            "method":"POST",
+            "authType":"USER",
+            "throttlingTier":"UNLIMITED",
```
Comment (alexec) [inline resources/public/api/definition.json]: Don't we want to throttle?

Comment (danmassie) [inline resources/public/api/definition.json]: It would be nice, but there is no throttling in the API platform. The only tier is UNLIMITED.

Comment (alexec) [inline resources/public/api/definition.json]: This does nothing?

Comment (danmassie) [inline resources/public/api/definition.json]: Seems like it at the moment, but is needed to produce a valid definition.json file.

**Pull request merged**